examples/coap-server.c

```diff
@@ -336,9 +342,17 @@ init_resources(coap_context_t *ctx) {
 
 static void
 fill_keystore(coap_context_t *ctx) {
-  static uint8_t key[] = "secretPSK";
-  size_t key_len = sizeof( key ) - 1;
-  coap_context_set_psk( ctx, "CoAP", key, key_len );
+  if (cert_file) {
+    coap_dtls_pki_t dtls_pki;
+    memset (&dtls_pki, 0, sizeof(dtls_pki));
+    dtls_pki.public_cert = cert_file;
+    dtls_pki.private_key = cert_file;
+    coap_context_set_pki(ctx, &dtls_pki);
+    if (key_defined)
+      coap_context_set_psk(ctx, "CoAP", key, key_length);
+  }
+  else
```

Comment:
Please enclose the else-branch with parentheses to avoid confusion when reading the code.